foreman.fc

```diff
@@ -34,14 +34,19 @@
 # Passenger non-SCL file contexts
 
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
-/usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/lib/ruby/site_ruby/1.8/x86_64-linux/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/libexec/passenger/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
Just to preempt any mistake, don't change this to lib64, it's correct!

    -rwxr-xr-x. root root system_u:object_r:passenger_exec_t:s0 /usr/lib/ruby/gems/1.8/gems/passenger-4.0.18/helper-scripts/prespawn